andino_navigation/package.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>andino_navigation</name>
+  <version>0.0.1</version>
+  <description>
+    Bring up nav2 package with Andino.
+  </description>
+  <maintainer email="olmerg@ekumenlabs.com">Olmer Garcia</maintainer>
+  <maintainer email="franco.c@ekumenlabs.com">Franco Cipollone</maintainer>
+  <license file="LICENSE">BSD Clause 3</license>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+  <build_depend>nav2_common</build_depend>
+
+  <build_depend>navigation2</build_depend>
```

Comment:
Not sure why you need it in build_depend. Both for `navigation2` and `nav2_common`